src/midgard/logging.cc

```diff
@@ -38,12 +38,9 @@ std::string TimeStamp() {
   std::chrono::duration<double> fractional_seconds =
       (tp - std::chrono::system_clock::from_time_t(tt)) + std::chrono::seconds(gmt.tm_sec);
   // format the string
-  std::string buffer("year/mo/dy hr:mn:sc.xxxxxx0");
+  std::string buffer("year/mo/dy hr:mn:sc.xxxxxx");
   snprintf(&buffer.front(), buffer.length(), "%04d/%02d/%02d %02d:%02d:%09.6f", gmt.tm_year + 1900,
            gmt.tm_mon + 1, gmt.tm_mday, gmt.tm_hour, gmt.tm_min, fractional_seconds.count());
```

Comment:
this should be reverted, it was just added: https://github.com/valhalla/valhalla/pull/3974/files